projects/plugins/boost/package.json

```diff
@@ -0,0 +1,116 @@
+{
+	"name": "jetpack-boost",
+	"version": "1.1.0",
+	"description": "Boost your WordPress site's performance, from the creators of Jetpack",
+	"directories": {
+		"test": "tests"
+	},
+	"dependencies": {
+		"@wordpress/components": "^14.1.2",
+		"@wordpress/element": "^3.1.1",
+		"jetpack-boost-critical-css-gen": "git+ssh://git@github.com/automattic/jetpack-boost-critical-css-gen.git#trunk"
+	},
+	"devDependencies": {
```

Comment:
Give your devDependencies a good look to see what isn't needed anymore. A few that jump out at me:

* `eslint` is handled at the monorepo level. You might move any additional eslint plugins into `tools/js-tools/package.json` too, to keep them in one place.
* Same for `prettier`.
* `danger` and `husky` I think are similarly repo-level tools?
* You might consider pushing all the puppeteer stuff into a subdirectory that's not part of the main pnpm workspace, like we did with `projects/plugins/jetpack/tests/e2e/`, so people not running tests don't have to download browser images and such that get pulled in for E2E testing.